packages/colorpickers/src/index.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+export * from './elements/ColorPicker';
```

Comment:
Should we do named exports to match all other packages? Wild-card exports can affect the ability to tree-shake effectively (especially with `sideEffects: false` provided)

Reply:
Yeah, updated 👍🏽 .